src/components/sales/OrderBill.vue

```diff
@@ -112,11 +62,11 @@
 					</colgroup>
 					<tr>
 						<td><strong>Total Units: </strong></td>
-						<td>4</td>
+						<td>{{order.order_item?.length}}</td>
```

Comment:
Omit '?' from this statement and all the preceding statements.

Reply:
this sign was added to avoid console errors in case of order_item being null. What do you suggest?

Reply:
change done